tko_hr_pt/__openerp__.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# coding: utf-8
+########################################################################################
+#
+# Copyright (C) ThinkOpen Solutions (<http://thinkopen.solutions>). All Rights Reserved
+# Copyright (C) 2004-2010 Tiny SPRL (<http://tiny.be>). All Rights Reserved
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3.0 of the License, or (at your option) any later version.
+# 
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+# 
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library.
+#
+########################################################################################
+{
+    'name': 'Human Resources PT',
+    'version': '1.064',
+    'category': 'Human Resources',
+    "sequence": 38,
+    'complexity': "normal",
+    'description': """
+    """,
+    'author':"ThinkOpen Solutions PT",
```

Comment:
Add `,Odoo Community Association (OCA)` (to allow searching for OCA modules in the apps site)